essential-documentation/contribute-to-appflowy/software-contributions/environment-setup/building-on-linux.md

```diff
@@ -42,8 +43,28 @@ sudo apt-get install curl build-essential libsqlite3-dev libssl-dev clang cmake
 yay -S curl base-devel sqlite openssl clang cmake ninja pkg-config gtk3 unzip
 ```
 {% endtab %}
+
+{%tab title="openSUSE Tumbleweed" %}
+```bash
+sudo zypper install -t pattern devel_basis
+sudo zypper install libopenssl-devel sqlite3-devel clang cmake ninja pkgconf-pkg-config pkgconf libgtk-3-0 curl zip unzip gtk3-devel
```

Comment:
is it necessary to install libopenssl? (sorry if that sounds dumb, but I already had it installed on my system...)